.github/workflows/run-unit-tests.yml

```
@@ -46,6 +46,7 @@ jobs:
       - name: Install & Start Docker
         if: env.INSTALL_DOCKER == '1'
         run: |
+          brew unlink colima
```

Could this work?
```
          if brew list colima &>/dev/null; then
            brew unlink colima
          fi
 ```

---

It's added by default.
https://github.com/actions/runner-images/issues/6216#issuecomment-1274318785

---

I agree that this is better to check beforehand. Github actions might change their protocol down the line, so doing this check would future-proof our code.

It also might be good to add a comment:

```suggestion
    # Unlink colima to upgrade to the latest version 
    if brew list colima &>/dev/null; then
          brew unlink colima
          # unlinking colima dependency: go
          brew unlink go@1.21
    fi
```